docs/porting-guide.rst

```diff
@@ -1,49 +1,311 @@
 
-* Package ``cbapi.example_helpers`` -> ``cbc_sdk.helpers``
-* Package ``cbapi.psc`` -> ``cbc_sdk``
-* Package ``cbapi.psc.alerts_query`` -> ``cbc_sdk.platform``
-* Package ``cbapi.psc.devices_query`` -> ``cbc_sdk.platform``
-* Package ``cbapi.psc.livequery`` -> ``cbc_sdk.audit_remediation``
-* Package ``cbapi.psc.livequery.models`` -> ``cbc_sdk.audit_remediation``
-* Package ``cbapi.psc.defense`` -> ``cbc_sdk.endpoint_standard``
-* Package ``cbapi.psc.defense.models`` -> ``cbc_sdk.endpoint_standard``
-* Package ``cbapi.psc.threathunter`` -> ``cbc_sdk.enterprise_edr``
-* Package ``cbapi.psc.threathunter.models`` -> ``cbc_sdk.enterprise_edr``
+Overview
+--------
+CBC SDK has changes to package names, folder structure, and functions. Import statements will need to change for the packages, modules, and functions listed in this guide.
 
-Code Changes
-------------
-**Helper Functions:** Replace all calls to ``get_cb_defense_object()``, ``get_cb_livequery_object()``,
-``get_cb_psc_object()``, and ``get_cb_threathunter_object()`` with ``get_cb_cloud_object()``.
+Package Name Changes
+--------------------
+A number of packages have new name equivalents in the CBC SDK. Endpoint Standard and Enterprise EDR have had parts replaced to use the most current API routes.
 
-**Audit/Remediation Queries:**
+Top-level Package Name Change
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The top-level package name has changed from CBAPI to CBC SDK.
 
-* Replace ``cb.query(sql_query)`` with ``cb.select(Run).where(sql=sql_query)``.
-* Replace ``cb.query_history(query_string)`` with ``cb.select(RunHistory).where(query_string)``.
-* On a query object, use the ``policy_id()`` method instead of ``policy_ids()``.  Only one policy ID can be specified.
++-----------------+--------------------+
+| CBAPI Name (old)| CBC SDK Name (new) |
++=================+====================+
+| ``cbapi.psc``   | ``cbc_sdk``        |
++-----------------+--------------------+
 
-**Base API Object**
+Product Name Changes
+^^^^^^^^^^^^^^^^^^^^
+Carbon Black Cloud product names have been updated in the SDK.
 
-The different API objects, ``CbDefenseAPI``, ``CbLiveQueryAPI``, ``CbPSCBaseAPI``, and ``CbThreatHunterAPI`` are
-replaced with ``CBCloudAPI``.  Import this object from the ``cbc_sdk`` package, i.e. ``from cbc_sdk import CBCloudAPI``.
++----------------------------+-------------------------------+
+| CBAPI Name (old)           | CBC SDK Name (new)            |
++============================+===============================+
+| ``cbapi.psc.defense``      | ``cbc_sdk.endpoint_standard`` |
++----------------------------+-------------------------------+
+| ``cbapi.psc.livequery``    | ``cbc_sdk.audit_remediation`` |
++----------------------------+-------------------------------+
+| ``cbapi.psc.threathunter`` | ``cbc_sdk.enterprise_edr``    |
++----------------------------+-------------------------------+
+| Does not exist in CBAPI    | ``cbc_sdk.platform``          |
```

Comment:
We did have the new device model in `cbapi.psc`

Reply:
I've updated this